pom.xml

```diff
@@ -329,6 +332,26 @@
         <version>${gson-extras.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>${jackson.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+
+      <dependency>
```

Comment:
I have a question. What is this one?

Reply:
building fails without this version pinning
```
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-dependency-convergence) @ zeppelin-server ---
[WARNING]
Dependency convergence error for jakarta.activation:jakarta.activation-api:1.2.1 paths to dependency are:
+-org.apache.zeppelin:zeppelin-server:0.12.0-SNAPSHOT
  +-org.glassfish.jersey.media:jersey-media-json-jackson:2.30
    +-com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.12.7
      +-jakarta.activation:jakarta.activation-api:1.2.1
and
+-org.apache.zeppelin:zeppelin-server:0.12.0-SNAPSHOT
  +-org.glassfish.jersey.core:jersey-server:2.30
    +-jakarta.xml.bind:jakarta.xml.bind-api:2.3.3
      +-jakarta.activation:jakarta.activation-api:1.2.2
```

Reply:
Aha ~ Thank you for confirming it.